README.md

```diff
@@ -19,10 +20,25 @@ which is also the one used by [AutoStitch](http://matthewalunbrown.com/autostitc
 Eigen, CImg and FLANN are header-only, to simplify the compilation on different platforms.
 CImg and libjpeg are only used to read and write images, so you can easily get rid of them.
 
+### Steps
+
+* Download Cmake and install the cmake
+* Download Eigen and unzip it in panorama/Eigen, please ensure that panorama/Eigen/Eigen/Dense exists.
```

Comment:
Definitely we should allow users to use Eigen in the system. 
So users should set eigen path by environment vars only if they want to download Eigen by themselves.